packaging/QueueNado.spec

```diff
@@ -28,8 +28,8 @@ cd 3rdparty
 unzip -u gtest-1.7.0.zip
 cd ..
 /usr/local/probe/bin/cmake -DCMAKE_CXX_COMPILER_ARG1:STRING=' -fPIC -Ofast -m64 -Wl,-rpath,. -Wl,-rpath -Wl,/usr/local/probe/lib -Wl,-rpath -Wl,/usr/local/probe/lib64 ' -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_CXX_COMPILER=/usr/local/probe/bin/g++
-make
-./UnitTestRunner
+make -j8
```

Comment:
I know there was a reason why we didn't make it "-j" from the beginning.

Do you remember? Is it any reason why we are not compiling as fast as possible?

Was it perhaps because we thought that at full parallelism the outcome was not deterministic?